### PR TITLE
feat(case visualization) Wrap case visu page

### PR DESCRIPTION
**Pull request by DumitruCorini**
covers [BPO-492](https://bonitasoft.atlassian.net/browse/BPO-492)

No links had to be changed.
NOTE : The snapshot version has a name that is too long, so to test this, you need to update name of the zip to remote '-SNAPSHOT'.

### CODE REVIEW

File: portaljs-pages/page-admin-case-visualization/src/index.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+        function getPortalUrl() {
+            var locationHref = window.location.href;
+            var indexOfPortal = locationHref.indexOf('/portal/');
+            if (indexOfPortal >= 0) {
+                return locationHref.substring(0, indexOfPortal);
+            }
+        };
+
+        let processId = getUrlParameter("id");
```
Comment (abirembaut) [inline portaljs-pages/page-admin-case-visualization/src/index.html]: maybe we should change the name of this parameter

File: portaljs-pages/page-admin-case-visualization/package.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "name": "page-admin-process-details",
```
Comment (abirembaut) [inline portaljs-pages/page-admin-case-visualization/package.json]: page-admin-case-visu

File: portaljs-pages/page-admin-case-visualization/src/index.html (hunk 1 above)
Comment (DumitruCorini) [reply portaljs-pages/page-admin-case-visualization/src/index.html]: Do you mean id or processId?

Comment (DumitruCorini) [reply portaljs-pages/page-admin-case-visualization/src/index.html]: If id, then we would also need to change the page.

File: portaljs-pages/page-admin-case-visu/resources/page.properties [hunk 3]
```diff
@@ -0,0 +1,5 @@
+displayName=Bonita Admin Case Visu
```
Comment (abirembaut) [inline portaljs-pages/page-admin-case-visu/resources/page.properties]: "Visualization" full can be used for the displayname

File: portaljs-pages/page-admin-case-visu/resources/page.properties [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+name=custompage_adminCaseVisuBonitaV1
+description=Default admin case visu
```
Comment (abirembaut) [inline portaljs-pages/page-admin-case-visu/resources/page.properties]: "Visualization" full can be used for the description

File: settings.gradle [hunk 5]
```diff
@@ -10,6 +10,7 @@ pluginManagement {
 include 'page-process-list'
 include 'portaljs-pages:page-user-details'
 include 'portaljs-pages:page-admin-process-details'
+include 'portaljs-pages:page-admin-case-visualization'
```
Comment (abirembaut) [inline settings.gradle]: visu

**Pull request merged**